text/__main__.py

```diff
@@ -2,6 +2,8 @@
 
 if __name__ == "__main__":
     import text.generator
+    import text.editor
     import sys
 
-    print(text.generator.get_text().encode().decode(sys.stdout.encoding))
\ No newline at end of file
+    piece_of_text = text.generator.get_text()
+    print(text.editor.get_printable_sentence(text.editor.get_ss_sentence(piece_of_text)).encode().decode(sys.stdout.encoding))
```

Comment:
Don't you think that this construction is too heavy? Maybe, we should cut this to several commented steps?